bin/workflows/pycbc_create_offline_search_workflow

```diff
@@ -213,6 +212,37 @@ statfiles = []
 statfiles += wf.setup_trigger_fitting(workflow, insps, hdfbank,
                                       final_veto_file, final_veto_name)
 
+# setup 3 detector coinc for the filtering jobs
+ctags = [tag, 'full3d']
```

Comment:
@tapaimarton One thing I think is not ideal here is that it hardcodes the 3-ifo combination and each two detector combination. It would be nice if we did not have to do this. 

Can I propose an alternate approach? We know which ifos have been requested to analyze. One can simply assume you want to analyze all of them (in the future we could provide flags to control this is needed). As such just take the list of ifos as input and do the complete combinatorics (i.e. nchoose2..n). The only thing you then need to control is *which* ifo for each of these is the pivot and the *fixed* ones. We could have that specified in the ini file as you've done, in which case, you can programmatically set the tags based on the ifo combination. Another solution which may be simpler and probably all we want anyway is to simply set a priority order. I.E. 
```
[workflow-coincidence]
timeslide-precedence = H1, L1, V1, K1, I1
```
This would translate to select the ifos being analyzed, and choose the first as "pivot" and the second as "fixed' preserving the order we give. 

@tapaimarton @spxiwh Does that sound reasonable?

Reply:
@ahnitz @tapaimarton any reason I am not assigned/reviewing this given that I was doing so for the gitlab issue?

Reply:
@tdent Sorry, I forgot to add you here when I assigned the gitlab issue to you.